Tasks/VsTest/outputstream.ts

```diff
@@ -0,0 +1,33 @@
+import * as stream from 'stream';
+import * as os from 'os';
+import * as tl from 'vsts-task-lib/task';
+
+export class StringErrorWritable extends stream.Writable {
+    private value: string = '';
+
+    constructor(options: any) {
+        super(options);
+    }
+
+    _write(data: any, encoding: string, callback: Function): void {
+        this.value += data;
```

Comment:
is data is string? if you are converting data in the next line -> why concatenating stream type to string?